### PR TITLE
kalker advisory for parsing panic

### PR DESCRIPTION
**Pull request by nyw0102**

### CODE REVIEW

File: crates/kalker/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+url = "https://github.com/PaddiM8/kalker/pull/153"
+categories = ["denial-of-service"]
+#related = ["CVE-2022-31100"]
+
+[versions]
+patched = [">=2.2.0"]
+```
+# Panic due to inappropriate string
+
+When trying to parse untrusted string through kalker, kalker may pancic,
```
Comment (bjorn3) [inline crates/kalker/RUSTSEC-0000-0000.md:15]: *panic

Comment (nyw0102) [reply crates/kalker/RUSTSEC-0000-0000.md]: Thank you! I revised the report!

**Comment by riking** [comment]
I don't think that a panic in a library not used outside a single application binary (no dependents on crates.io) deserves an advisory. There's no indications that this is used as a networked service (and if only the developers host it as a network service, it can be handled internally).

In other words, use binary-focused advisory systems like CVE and customer mailing lists for this (and perhaps not even that, it's just a user input panic).

**Comment by Shnatsel** [comment]
I agree with @riking, so I'm going to go ahead and close this.

Panics are generally not advisory-worthy on their own, and this is not a high-stakes environment like a networking server where it could potentially matter.

**Pull request closed without merging**